### DISCUSSION TITLE
How should websites with toggleable dark mode be handled?

### DISCUSSION THREAD

**Discussion by PyroGenesis** [darkreader/darkreader#9727] [Q&A] [answered]
If a website has a toggleable dark mode (let's say initially it is light mode), how is Dark Reader supposed to handle it?

1. Consider it to be a dark-mode website, and add it to `dark-sites.config`
2. Consider it to just be a normal light-mode website and let the Dynamic Theme handle it (with any fixes added to `dynamic-theme-fixes.config`). Users will then disable Dark Reader manually if they wish to use the website dark mode.

Example:
https://docs.aws.amazon.com/workspaces/latest/adminguide/amazon-workspaces.html
You can switch to dark mode using the Preferences option in the top-right

I already have a hunch but I just want to confirm the appropriate action for this scenario.

**Comment by Gusted** [comment, marked as answer]
HI @PyroGenesis 

For those that are initially light-mode, they are just light-by-default and shouldn't be added to the dark-sites.config list. Dark Reader will add fixes for them in the config if needed. 

> Users will then disable Dark Reader manually if they wish to use the website dark mode.

That's indeed the expected workflow, or users should use the _experimental_ feature that detects dark mode.
